inc/atm_def.h

```diff
@@ -1,17 +1,22 @@
 #ifndef ATM_DEF_H
 #define ATM_DEF_H
 
-#define MAX_DEPOSITS	10000
-#define MIN_DEPOSITS	10
+#define MAX_DEPOSITS		10000
+#define MIN_DEPOSITS		10
 
-#define MIN_WITHDRAW	MIN_DEPOSITS
-#define MAX_WITHDRAW	1000
+#define MIN_WITHDRAW		MIN_DEPOSITS
+#define MAX_WITHDRAW		1000
+
+//#define FLAG_UPDATE_PINS	0b00001111
+//#define FLAG_UPDATE_BALA	0b11110000
```

Comment:
Is this needed or am I getting too fancy for no reason? My goal is to limit # of unnecessary Write operations

Reply:
Hmmm...